lib/identity_config.rb

```diff
@@ -264,6 +264,7 @@ def self.build_store(config_map)
     config.add(:in_person_send_proofing_notifications_enabled, type: :boolean)
     config.add(:in_person_stop_expiring_enrollments, type: :boolean)
     config.add(:include_slo_in_saml_metadata, type: :boolean)
+    config.add(:invalid_gpo_confirmation_zipcode, type: :string)
```

Comment:
My thinking was that I don't want to even have this available in prod and we _tend_ to prefer explicit config flags rather than `unless Rails.env.prod?` type stuff.